helm/mds/templates/ingress.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   http:
   - match:
     - uri:
-        prefix: {{ $api.pathPrefix }}
+        regex: {{ printf "^%s[^-][/]{0,1}.*$" $api.pathPrefix }}
```

Comment:
I trust your `regex`, but this would also work, I think:
```
(^[/]policy$|^[/]policy[/].*)
```

Reply:
Do you need to escape the `/`? something like `[^-][\/]{0,1}.*`

Reply:
> Do you need to escape the `/`? something like `[^-][\/]{0,1}.*`

You'd think so, and I did have to in regex101.com, but here nope.  Confirmed on sandbox.

Reply:
> I trust your `regex`, but this would also work, I think:
> 
> ```
> (^[/]policy$|^[/]policy[/].*)
> ```

Looks like it'll also work without the character groupings around the forward-slashes - I like it @drtyh2o - much more obvious what it's doing.

`(^/policy$|^/policy/.*)`